Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -260,8 +260,9 @@ private void UpdateHitTarget()
 
                     if (isHit)
                     {
-                        LookedAtTarget = hitInfo.collider.gameObject;
-                        LookedAtEyeTarget = LookedAtTarget.GetComponent<EyeTrackingTarget>();
+                        LookedAtEyeTarget = hitInfo.collider.transform.GetComponent<EyeTrackingTarget>() ??
+                        hitInfo.collider.transform.GetComponentInChildren<EyeTrackingTarget>();
+                        LookedAtTarget = LookedAtEyeTarget?.gameObject;
                         LookedAtPoint = hitInfo.point;
```

Comment:
Hi @peader , Thanks for contributing to MRTK and also adding a test for your proposal.
I am thinking how could we handle a scenario where other hit parent children also have the EyeTrackingTarget component attached to?

Reply:
Hi @vaoliva , could you give me an example use case for such a feature?